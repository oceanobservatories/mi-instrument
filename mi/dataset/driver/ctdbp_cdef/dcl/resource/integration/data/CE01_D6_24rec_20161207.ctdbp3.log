2016/12/07 00:28:05.057 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 00:30:12.064 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 00:30:26.693  12.0247,  3.57278,    0.999, 617, 118, 68, 07 Dec 2016 00:30:20
2016/12/07 00:31:01.889 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 01:28:04.388 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 01:30:03.096 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 01:30:26.530  12.0527,  3.57986,    1.003, 597, 120, 69, 07 Dec 2016 01:30:20
2016/12/07 01:31:01.732 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 02:28:03.867 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 02:30:11.783 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 02:30:26.425  12.0492,  3.58244,    1.010, 617, 121, 70, 07 Dec 2016 02:30:20
2016/12/07 02:31:01.619 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 03:28:05.788 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 03:30:13.549 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 03:30:28.171  12.0498,  3.58513,    1.059, 644, 117, 70, 07 Dec 2016 03:30:21
2016/12/07 03:31:03.374 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 04:28:04.995 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 04:30:13.195 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 04:30:27.832  12.0665,  3.58784,    1.024, 669, 114, 71, 07 Dec 2016 04:30:21
2016/12/07 04:31:03.030 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 05:28:03.660 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 05:30:03.810 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 05:30:27.267  12.0614,  3.58820,    1.012, 691, 115, 70, 07 Dec 2016 05:30:20
2016/12/07 05:31:02.460 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 06:28:05.032 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 06:30:12.437 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 06:30:27.059  12.0192,  3.58241,    1.059, 682, 116, 70, 07 Dec 2016 06:30:20
2016/12/07 06:31:02.252 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 07:28:04.314 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 07:30:03.837 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 07:30:27.271  12.0405,  3.58507,    1.071, 657, 117, 70, 07 Dec 2016 07:30:20
2016/12/07 07:31:02.474 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 08:28:04.980 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 08:30:08.311 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 08:30:31.728  12.0063,  3.58220,    0.993, 575, 117, 70, 07 Dec 2016 08:30:25
2016/12/07 08:31:06.914 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 09:28:05.976 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 09:30:13.696 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 09:30:28.322  12.0170,  3.58387,    0.998, 557, 116, 70, 07 Dec 2016 09:30:22
2016/12/07 09:31:03.766 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 10:28:05.840 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 10:30:04.174 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 10:30:27.609  11.9930,  3.58195,    1.074, 544, 113, 69, 07 Dec 2016 10:30:21
2016/12/07 10:31:02.803 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 11:28:03.589 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 11:30:02.723 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 11:30:26.141  11.9451,  3.57698,    1.054, 675, 115, 69, 07 Dec 2016 11:30:19
2016/12/07 11:31:01.328 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 12:28:04.448 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 12:30:03.214 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 12:30:26.668  11.9255,  3.57500,    1.033, 623, 112, 70, 07 Dec 2016 12:30:20
2016/12/07 12:31:01.862 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 13:28:03.807 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 13:30:03.707 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 13:30:27.153  11.9251,  3.57544,    1.078, 649, 117, 69, 07 Dec 2016 13:30:20
2016/12/07 13:31:02.348 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 14:28:02.315 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 14:30:02.566 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 14:30:25.982  11.9351,  3.57924,    1.072, 653, 113, 69, 07 Dec 2016 14:30:19
2016/12/07 14:31:01.181 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 15:28:04.865 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 15:30:03.554 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 15:30:26.982  11.9151,  3.57946,    1.060, 629, 110, 70, 07 Dec 2016 15:30:20
2016/12/07 15:31:02.180 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 16:28:04.692 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 16:30:11.362 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 16:30:25.996  11.8948,  3.57896,    1.044, 606, 114, 71, 07 Dec 2016 16:30:19
2016/12/07 16:31:01.187 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 17:28:04.754 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 17:30:03.145 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 17:30:26.593  11.8943,  3.57987,    1.042, 606, 100, 71, 07 Dec 2016 17:30:20
2016/12/07 17:31:01.792 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 18:28:04.503 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 18:30:02.799 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 18:30:26.209  11.9468,  3.58655,    1.036, 566, 100, 68, 07 Dec 2016 18:30:19
2016/12/07 18:31:01.405 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 19:28:03.933 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 19:30:02.577 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 19:30:25.996  11.9716,  3.58918,    1.030, 530, 102, 66, 07 Dec 2016 19:30:19
2016/12/07 19:31:01.194 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 20:28:05.772 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 20:30:12.203 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 20:30:26.850  11.9799,  3.59077,    1.024, 634, 104, 67, 07 Dec 2016 20:30:20
2016/12/07 20:31:02.039 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 21:28:05.707 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 21:30:12.812 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 21:30:27.448  11.9194,  3.58517,    1.001, 571, 113, 68, 07 Dec 2016 21:30:21
2016/12/07 21:31:02.650 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 22:28:03.456 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 22:30:02.767 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/12/07 22:30:26.224  11.9071,  3.58623,    0.989, 558, 121, 70, 07 Dec 2016 22:30:19
2016/12/07 22:31:01.410 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/12/07 23:28:05.759 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/12/07 23:30:13.623 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/12/07 23:30:28.262  11.9049,  3.58800,    0.972, 525, 128, 69, 07 Dec 2016 23:30:21
2016/12/07 23:31:03.454 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
