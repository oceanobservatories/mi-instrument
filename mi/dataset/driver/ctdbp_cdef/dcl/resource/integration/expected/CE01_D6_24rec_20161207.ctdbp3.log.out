
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690059426.6929998398 |          6 |         3690059420 | 3690059426.6929998398 |      port_timestamp |           ok |      3.57278 |             null |                     null |    0.999 | 12.0247
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered |  3690063026.529999733 |          6 |         3690063020 |  3690063026.529999733 |      port_timestamp |           ok |      3.57986 |             null |                     null |    1.003 | 12.0527
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690066626.4250001907 |          6 |         3690066620 | 3690066626.4250001907 |      port_timestamp |           ok |      3.58244 |             null |                     null |     1.01 | 12.0492
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690070228.1710000038 |          6 |         3690070221 | 3690070228.1710000038 |      port_timestamp |           ok |      3.58513 |             null |                     null |    1.059 | 12.0498
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690073827.8319997787 |          6 |         3690073821 | 3690073827.8319997787 |      port_timestamp |           ok |      3.58784 |             null |                     null |    1.024 | 12.0665
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690077427.2670001984 |          6 |         3690077420 | 3690077427.2670001984 |      port_timestamp |           ok |       3.5882 |             null |                     null |    1.012 | 12.0614
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690081027.0590000153 |          6 |         3690081020 | 3690081027.0590000153 |      port_timestamp |           ok |      3.58241 |             null |                     null |    1.059 | 12.0192
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690084627.2709999084 |          6 |         3690084620 | 3690084627.2709999084 |      port_timestamp |           ok |      3.58507 |             null |                     null |    1.071 | 12.0405
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690088231.7279996872 |          6 |         3690088225 | 3690088231.7279996872 |      port_timestamp |           ok |       3.5822 |             null |                     null |    0.993 | 12.0063
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690091828.3220000267 |          6 |         3690091822 | 3690091828.3220000267 |      port_timestamp |           ok |      3.58387 |             null |                     null |    0.998 |  12.017
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered |  3690095427.609000206 |          6 |         3690095421 |  3690095427.609000206 |      port_timestamp |           ok |      3.58195 |             null |                     null |    1.074 |  11.993
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered |  3690099026.140999794 |          6 |         3690099019 |  3690099026.140999794 |      port_timestamp |           ok |      3.57698 |             null |                     null |    1.054 | 11.9451
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690102626.6680002213 |          6 |         3690102620 | 3690102626.6680002213 |      port_timestamp |           ok |        3.575 |             null |                     null |    1.033 | 11.9255
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690106227.1529998779 |          6 |         3690106220 | 3690106227.1529998779 |      port_timestamp |           ok |      3.57544 |             null |                     null |    1.078 | 11.9251
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered |  3690109825.982000351 |          6 |         3690109819 |  3690109825.982000351 |      port_timestamp |           ok |      3.57924 |             null |                     null |    1.072 | 11.9351
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered |  3690113426.982000351 |          6 |         3690113420 |  3690113426.982000351 |      port_timestamp |           ok |      3.57946 |             null |                     null |     1.06 | 11.9151
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690117025.9960002899 |          6 |         3690117019 | 3690117025.9960002899 |      port_timestamp |           ok |      3.57896 |             null |                     null |    1.044 | 11.8948
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690120626.5929999352 |          6 |         3690120620 | 3690120626.5929999352 |      port_timestamp |           ok |      3.57987 |             null |                     null |    1.042 | 11.8943
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690124226.2090001106 |          6 |         3690124219 | 3690124226.2090001106 |      port_timestamp |           ok |      3.58655 |             null |                     null |    1.036 | 11.9468
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690127825.9960002899 |          6 |         3690127819 | 3690127825.9960002899 |      port_timestamp |           ok |      3.58918 |             null |                     null |     1.03 | 11.9716
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690131426.8499999046 |          6 |         3690131420 | 3690131426.8499999046 |      port_timestamp |           ok |      3.59077 |             null |                     null |    1.024 | 11.9799
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690135027.4479999542 |          6 |         3690135021 | 3690135027.4479999542 |      port_timestamp |           ok |      3.58517 |             null |                     null |    1.001 | 11.9194
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690138626.2239999771 |          6 |         3690138619 | 3690138626.2239999771 |      port_timestamp |           ok |      3.58623 |             null |                     null |    0.989 | 11.9071
 CE01ISSM | SBD17 | 06-CTDBPC000 | 3690057600 | telemetered | 3690142228.2620000839 |          6 |         3690142221 | 3690142228.2620000839 |      port_timestamp |           ok |        3.588 |             null |                     null |    0.972 | 11.9049

(24 rows)
